pyinjective/async_client.py

```diff
@@ -173,8 +173,11 @@ async def close_chain_channel(self):
         self.cron.stop()
 
     async def sync_timeout_height(self):
-        block = await self.get_latest_block()
-        self.timeout_height = block.block.header.height + DEFAULT_TIMEOUTHEIGHT
+        try:
+            block = await self.get_latest_block()
+            self.timeout_height = block.block.header.height + DEFAULT_TIMEOUTHEIGHT
+        except Exception:
```

Comment:
looks good, let's log debug here, if people need to see it, they can change the log level